File: app/src/main/java/app/wt/noolis/model/CategoryIcon.java
package app.wt.noolis.model;

import java.io.Serializable;

/**
 * Created by Kodok on 14/06/2016.
 */
public class CategoryIcon implements Serializable {
    private String icon;
    private String color;
    private boolean checked;

    public CategoryIcon() {
    }

    public CategoryIcon(String icon, String color) {
        this.icon = icon;
        this.color = color;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public boolean isChecked() {
        return checked;
    }

    public void setChecked(boolean checked) {
        this.checked = checked;
    }
}

File: app/src/main/java/app/wt/noolis/model/CategoryIcon.kt
package app.wt.noolis.model

import java.io.Serializable

/**
 * Created by Kodok on 14/06/2016.
 */
class CategoryIcon : Serializable {
    var icon: String? = null
    var color: String? = null
    var isChecked: Boolean = false

    constructor() {}

    constructor(icon: String, color: String) {
        this.icon = icon
        this.color = color
    }
}
